packages/web-config-server/src/aggregator/buildAggregationOptions.js

```diff
@@ -28,12 +28,14 @@ export const buildAggregationOptions = async (
     aggregationOrder: entityAggregationOrder = DEFAULT_ENTITY_AGGREGATION_ORDER,
   } = entityAggregationOptions;
 
-  // Note aggregationType and aggregationConfig might be undefined
-  const inputAggregations = aggregations || [{ type: aggregationType, config: aggregationConfig }];
+  // Note aggregationType and aggregationConfig might be undefined, in which case it will default
+  // to MOST_RECENT if no multi-aggregations are provided
```

Comment:
I had a logic change in here originally that filtered out `undefined` single aggregations, but it turns out that was required so that it would default to `MOST_RECENT`. Have updated the comment to clarify, and these changes should now be superficial only